backend-rust/migrations/0003_release_schedule.up.sql

```diff
@@ -0,0 +1,31 @@
+-- Migration script for adding the scheduled release table.
```

Comment:
I think we should stick to one sql file for now, and only do migrations when we have to, since we will nuke the database for now anyway.